src/astro/sql/operators/append.py

```diff
@@ -22,14 +27,19 @@ def __init__(
         self,
         source_table: Table,
         target_table: Table,
-        source_to_target_columns_map: Optional[Dict[str, str]] = None,
+        columns: APPEND_COLUMN_TYPE = None,
         task_id: str = "",
         **kwargs: Any,
     ) -> None:
         self.source_table = source_table
         self.target_table = target_table
-        self.source_to_target_columns_map = source_to_target_columns_map or {}
-
+        if isinstance(columns, (list, tuple)):
```

Comment:
Hey @kaxil, nothing major, but can we move lines 36 - 41 to a function and use that function in the append and merge operators just to avoid some code duplication?

Reply:
In this case, [DAMP](https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests) (“Descriptive and Meaningful Phrases”) approach is better than DRY (“Don’t Repeat Yourself”) as they are hardly 4-5 lines :)